### PR TITLE
Add io.github.swordpuffin.hunt

### PR DESCRIPTION
**Pull request by SwordPuffin**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 
An easy to play word search game made with Python and a GTK4 UI.
See: https://github.com/SwordPuffin/Hunt

- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. 

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Hunt
+
+A simple word search game built with GTK4 and Python.
+
+
+## How To Play
```
Comment (hfiguiere) [inline README.md:6]: just to set expectation: no user will ever see this.

File: io.github.swordpuffin.hunt.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.swordpuffin.hunt",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "hunt",
+    "finish-args" : [
+        "--share=network",
```
Comment (hfiguiere) [inline io.github.swordpuffin.hunt.json:8]: does this really need network access?

**Comment by hfiguiere** [comment]
bot, build io.github.swordpuffin.hunt

File: io.github.swordpuffin.hunt.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "modules" : [
+        {
+            "name" : "hunt",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/SwordPuffin/Hunt.git",
+                    "tag" : "v1.0.1"
```
Comment (bbhtt) [inline io.github.swordpuffin.hunt.json:34]: screenshots need to be direct links, not links to HTML pages https://github.com/SwordPuffin/Hunt/blob/fadf41e33832cb47bac84e48b79e73af08574a7b/data/io.github.swordpuffin.hunt.metainfo.xml.in#L24-L27

https://raw.githubusercontent.com/SwordPuffin/Hunt/refs/tags/v1.0.1/data/screenshots/Screenshot1.png
https://raw.githubusercontent.com/SwordPuffin/Hunt/refs/tags/v1.0.1/data/screenshots/Screenshot2.png

**Comment by SwordPuffin** [comment]
bot, build io.github.swordpuffin.hunt

**Pull request closed without merging**